src/main/scala/gitbucket/core/util/Validations.scala

```diff
@@ -45,6 +45,20 @@ trait Validations {
       }
   }
 
+  /**
+   * Constraint for the milestone identifier.
+   */
+  def milestone: Constraint = new Constraint() {
```

Comment:
Why do you add this constraint to the milestone? Also, if this constraint is used by only `milestoneForm`, it should be defined in `MilestonesController.scala` as same as `uniqueMilestone` constraint.

Reply:
@takezoe I just thought repository has this constraint and so I will add it to milestone as well. I will just remove this constraint.